Automattic/jetpack
# Boost: Remove global state from Hero, Tabs and Table components

## Proposed changes:

* Moves global state to parent component in favor of props;

### Other information:


## Jetpack product discussion



pc9hqz-286-p2

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Make sure the "Hero" section of the Image Size Analysis works as expected.
* Request a new report and open the detailed page - it should dynamically update as the report progresses.
![image](https://github.com/Automattic/jetpack/assets/11799079/6415b535-ca5a-49bf-985d-98f84a9415d5)


* Make sure the "Tabs" section of the Image Size Analysis works as expected.
![CleanShot 2023-09-15 at 17 49 17](https://github.com/Automattic/jetpack/assets/11799079/ac684837-1367-46aa-9aa7-b77e56fb2962)


* Make sure the "Table" section of the Image Size Analysis works as expected.
![CleanShot 2023-09-15 at 17 48 24](https://github.com/Automattic/jetpack/assets/11799079/1145e3b9-041a-4026-b43d-8197bee8eb63)
